rpcs3/Emu/RSX/Common/ProgramStateCache.h

```diff
@@ -149,7 +151,8 @@ class program_state_cache
 			return std::forward_as_tuple(I->second, true);
 		}
 		LOG_NOTICE(RSX, "VP not found in buffer!");
-		vertex_program_type& new_shader = m_vertex_shader_cache[rsx_vp];
+
```

Comment:
Don't add unnecessary formatting. If it doesn't change functionality, leave it alone